.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+<!-- Hi there! Please use the template below as a guide for opening a pull request.
+This template is designed to help the pull request process go smoother but if anything doesn't make sense to you, please open the pull request anyway and we will sort it out. Thank you for your contribution!! -->
+
+### Description of the proposed additions or changes
+<!-- If the additions or changes aren't self explanatory feel free to leave a description or notes here. -->
+
+
+### Related issues
+<!-- GitHub will link issues to pull requests if you list the related issue numbers like "#2, #17". -->
+
+
+### People tagged for review
```

Comment:
```suggestion
### Suggested reviewers (optional)
```

All the fields are optional, but this one seems even more optional than the others.  Is that worth noting?